### PR TITLE
Fix broken guidelines link from expander page

### LINKED ISSUE #742: Guidelines link in expander page leads to 404
**Describe the bug**

**Steps to reproduce the bug**

1. Go to expander page
2. click guidelines
3. 404

**Expected behavior**

Link doesn't work, so it should be removed.

**Actual behavior**

Broken link.

**Screenshots**

**Version Info**
- [x] App downloaded from Store 
    - If so, which version? (Found on the Settings page of the app.) latest
- [x] App built locally from source 
  - If so, which branch? (WinUI 3 versions are built off the winui3preview branch) latest version of master

**Additional context**

Windows 10 version:
- [ ] Insider Build (xxxxx)
- [ ] May 2020 Update (19041)
- [ ] November 2019 Update (18363)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

### PR DESCRIPTION
**Pull request by winston-de**
## Description

Removed broken expander guidelines link.

## Motivation and Context

closes #742

## How Has This Been Tested?

manually
## Screenshots (if appropriate):
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by winston-de** [comment]
Also, the guidance link in the pips pager page also leads to a 404. Adding here because it doesn't seem worth opening another issue and pr for.
image

File: XamlControlsGallery/DataModel/ControlInfoData.json [hunk 1]
```diff
@@ -1240,10 +1240,6 @@
             {
               "Title": "PipsPager - API",
               "Uri": "https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.pipspager"
-            },
-            {
-              "Title": "Guidance",
-              "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/pipspager"
```
Comment (chingucoding) [inline XamlControlsGallery/DataModel/ControlInfoData.json:1246]: @anawishnoff @gabbybilka @mikehillberg I wasn't able to find any "pattern" link for pipspager, is this intentional?

File: XamlControlsGallery/DataModel/ControlInfoData.json [hunk 2]
```diff
@@ -1372,10 +1368,6 @@
             {
               "Title": "Expander - API",
               "Uri": "https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.expander"
-            },
-            {
-              "Title": "Guidelines",
-              "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/expander"
```
Comment (chingucoding) [inline XamlControlsGallery/DataModel/ControlInfoData.json:1378]: @anawishnoff @kat-y @mikehillberg I wasn't able to find any "pattern" link for expander, is this intentional?

**Comment by gabbybilka** [comment]
Ah, here are the guidance links. We had a bit of a doc shuffle :)

- https://docs.microsoft.com/en-us/windows/apps/design/controls/pipspager
- https://docs.microsoft.com/en-us/windows/apps/design/controls/expander

**Comment by winston-de** [comment]
> Ah, here are the guidance links. We had a bit of a doc shuffle :)
> 
> * https://docs.microsoft.com/en-us/windows/apps/design/controls/pipspager
> * https://docs.microsoft.com/en-us/windows/apps/design/controls/expander

Thanks, I updated the pr.

**Comment by winston-de** [comment]
Quick question though, the pips pager link is labeled guidance, while the expander link is labeled guidelines. Should the labels for 0one/both be updated?

**Comment by gabbybilka** [comment]
@anawishnoff I'll leave the final approval for these changes up to you, LGTM!

File: XamlControlsGallery/DataModel/ControlInfoData.json [hunk 3]
```diff
@@ -1957,7 +1958,7 @@
               "Uri": "https://www.docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.animatedicon"
             },
             {
-              "Title": "AnimatedIcon - Guidance",
+              "Title": "AnimatedIcon - Guidelines ",
```
Comment (chingucoding) [inline XamlControlsGallery/DataModel/ControlInfoData.json:1961]: For consistency, should this just be "Guidelines"?

Comment (winston-de) [reply XamlControlsGallery/DataModel/ControlInfoData.json:1961]: Good point, but should it be reversed? Other links seem to follow that format, eg "AnimatedIcon - API".

Comment (chingucoding) [reply XamlControlsGallery/DataModel/ControlInfoData.json:1961]: It seems that all controls just have "Guidelines" while the other links, such as the API links, are name "[control name] API" etc.

Comment (anawishnoff) [reply XamlControlsGallery/DataModel/ControlInfoData.json:1961]: I just took a quick look through the gallery and I think most pages have just "Guidelines", so leaving it at that would probably be most consistent. But your point is valid that there's inconsistency between the formatted API link and guidance link. That might be a good thing to implement as a larger change.

**Review by chingucoding — APPROVED** [review]
LGTM, thank you @winston-de

Also, great branch name! 😄

**Review by anawishnoff — APPROVED** [review]
Thanks @winston-de!

**Pull request merged**